### PR TITLE
@craigspaeth - Add prundupify to asset build

### PR DESCRIPTION
**Pull request by broskoski**
Sigh, so after banging my head against this ( https://github.com/artsy/force/issues/1380 ) for two days, I found this issue: https://github.com/substack/factor-bundle/issues/51 . Apparently factor-bundle's de-dupe feature is incompatible with our Browserify. Someone in this thread made a browserify plugin which de-dupe's the required modules ( https://github.com/tellnes/prundupify ). 

I did a cursory comparison, and it doesn't look like file sizes are much different, but you might know whether this is a bad idea or not. It _does_ however fix the issue that I was getting with importing reaction-force client-side in production.

### CODE REVIEW

**Comment by craigspaeth** [comment]
LGTM! That's an unfortunate issue with factor-bundle, but I'm glad you discovered this plugin. Sorry for the grief!

File: desktop/assets/categories.coffee [hunk 1]
```diff
@@ -13,7 +13,7 @@ routes =
 
   '''
   /tag
-  ''': require('../apps/tag/client.coffee').init
+  ''': require('../apps/tag/client.js').default.init
```
Comment (alloy) [inline desktop/assets/categories.coffee]: Is there a need to include file extensions when using CoffeeScript? With vanilla JS there’s no need as extensions are registered with `require`.

**Comment by alloy** [comment]
Great job finding the issue, @broskoski 👨‍🎤

File: desktop/assets/categories.coffee (hunk 1 above)
Comment (craigspaeth) [inline desktop/assets/categories.coffee]: By default yes, but you can [configure this in browserify](https://github.com/substack/node-browserify#browserifyfiles--opts) with `extensions`.

Comment (alloy) [inline desktop/assets/categories.coffee]: 👍

**Pull request merged**